resolwe_bio/tools/star_sj_to_bed12.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
+"""Recalculate from STAR SJ.out.tab file to BED12 format."""
+
+import argparse
+
+import numpy as np
+import pandas as pd
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('-f', '--sj_file', help="STAR SJ.out.tab output file")
+args = parser.parse_args()
+
+sj_file = pd.read_csv(args.sj_file, delimiter='\t', header=None)
```

Comment:
Would it make sense to also document the content/columns of SJ file?

Reply:
I don't know if is more understandable or not, but I have added it.